pkg/machine/hyperv/hutil.go

```diff
@@ -0,0 +1,30 @@
+//go:build windows
+
+package hyperv
+
+import (
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/windows"
+)
+
+func HasHyperVAdminRights() bool {
```

Comment:
If it's not too hard, it would be helpful to add a test for this (that's not a blocker).